django/contrib/auth/__init__.py

```diff
@@ -82,6 +82,8 @@ def authenticate(request=None, **credentials):
 
 
 def _authenticate_with_backend(backend, backend_path, request, credentials):
+    # Prevent credentials mutation from propagating.
+    credentials = dict(credentials)
```

Comment:
how about using {} in place of dict?

Reply:
Do you mean a comprehension, @auvipy?

That would only be making a shallow copy, which `dict()` does above.

However, I'd suggest `credentials.copy()` as it is more explicit.